cryptoinvestor/models/user.py

```diff
@@ -7,3 +8,30 @@ def __init__(self, *, username: str, password: str, account: Account):
         self.username = username
         self.password = password
         self.account = account
+        self.id = id
+        self.rates = {}
+        self.coin_status = 10000
+        self.bought = dict()
+
+    def buy(self,count, price, crypto_name) -> bool:
+        if self.coin_status >= (count * price):
+            if self.bought.get(crypto_name) is None:
+                self.bought[crypto_name] = count
+            else:
+                self.bought[crypto_name] += count
+            self.coin_status -= count * price
+           # print("coins: ",self.coin_status," bought: ",self.bought)
+        else:
+           # print("You have not enough money")
+            return False
+        return True
+
+    def sell(self,count, price, crypto_name) -> bool:
+        if self.bought.get(crypto_name) is None or self.bought[crypto_name] == 0:
```

Comment:
You can do this in one go like:

```python
if self.bought.get(crypto_name, 0) == 0:
...
```

Reply:
But this means you can't sell a currency only if you haven't bought it. But with this you can sell more than you have. That is not correct behavior. We should also write tests for this.